### PR TITLE
Add public and private key to import fields

### LINKED ISSUE #53: Imports: Add public and private key
```php
Field::make('product_line')
```
```php
Field::make('product_line', 'productLine')
```

Lets replace this work around I added [here](https://github.com/headlesslaravel/formations/pull/54) with a internal key matching [this section](https://github.com/headlesslaravel/formations/blob/main/src/Filter.php#L121-L145) in filter.php.. the first parameter being the public facing key, in filters thats the url parameter, and the internal key being the relationship name.. sometimes they are the same but when the format is different.. the extra parameter is useful.

### PR DESCRIPTION
**Pull request by mitulgolakiya**
closes #53

### CODE REVIEW

File: src/Imports/Import.php [hunk 1]
```diff
@@ -83,11 +84,16 @@ public function getReplacements(Collection $rows): array
...
+        /** @var Field[] $relations */
         $relations = collect($this->fields)->filter->isRelation();
 
         // author, category, etc
         foreach ($relations as $relation) {
-            $relationshipName = Str::camel($relation->key);
+            if (!empty($relation->internal)) {
+                $relationshipName = $relation->internal;
+            } else {
+                $relationshipName = Str::camel($relation->key);
```
Comment (dillingham) [inline src/Imports/Import.php:95]: Let's remove this camel incase they want the relation in a different format.

And add the if internal is null check to `init` and setting $internal if null

Comment (dillingham) [reply src/Imports/Import.php:95]:
```php
Field::make('category_name', 'category.name')
```

Comment (dillingham) [reply src/Imports/Import.php:95]: This might effect #65. Sorry it wasnt more clear

**Review by dillingham — CHANGES_REQUESTED** [review]

File: src/Imports/Import.php (hunk 1 above)
Comment (mitulgolakiya) [reply src/Imports/Import.php:95]: Sorry @dillingham not able to understand the comment here :(

Comment (dillingham) [reply src/Imports/Import.php:95]: @mitulgolakiya no problem :) I was half awake when I explained it

Here was what I was going for: https://github.com/headlesslaravel/formations/pull/67/files

**Pull request merged**